code/obj/item/storage/med_chem.dm

```diff
@@ -71,29 +71,48 @@
-	spawn_contents = list(/obj/item/reagent_containers/syringe/antitoxin = 3,\
-	/obj/item/reagent_containers/pill/antitox = 3,\
+	spawn_contents = list(/obj/item/reagent_containers/emergency_injector/epinephrine,\
+	/obj/item/reagent_containers/syringe/calomel,\
+	/obj/item/reagent_containers/pill/antirad = 2,\
+	/obj/item/reagent_containers/pill/antitox = 2,\
 	/obj/item/device/analyzer/healthanalyzer)
 
 /obj/item/storage/firstaid/oxygen
 	name = "oxygen deprivation first aid"
 	icon_state = "O21"
 	item_state = "O21"
-	desc = "A first aid kit that contains four pills of salbutamol, which is able to counter injuries caused by suffocation. Also contains a health analyzer to determine the health of the patient."
+	desc = "A first aid kit that contains three pills and two auto-injectors of salbutamol, which is able to counter injuries caused by suffocation. Also contains a health analyzer to determine the health of the patient."
 	kit_styles = list("O21", "O22", "O23", "O24")
-	spawn_contents = list(/obj/item/reagent_containers/pill/salbutamol = 4,\
+	spawn_contents = list(/obj/item/reagent_containers/pill/salbutamol = 3,\
+	/obj/item/reagent_containers/emergency_injector/salbutamol = 2,\
+	/obj/item/reagent_containers/emergency_injector/epinephrine,\
 	/obj/item/device/analyzer/healthanalyzer)
 
 /obj/item/storage/firstaid/brain
 	name = "neurological damage first aid"
 	icon_state = "brain1"
 	item_state = "brain1"
-	desc = "A medical kit that contains four pills of mannitol, which can heal brain damage. Also contains a health analyzer to determine the health of the patient."
+	desc = "A medical kit that contains three pills and two auto-injectors of mannitol, which can heal brain damage. Also contains a health analyzer to determine the health of the patient."
 	kit_styles = list("brain1", "brain2", "brain3")
-	spawn_contents = list(/obj/item/reagent_containers/pill/mannitol = 4,\
+	spawn_contents = list(/obj/item/reagent_containers/pill/mannitol = 3,\
+	/obj/item/reagent_containers/emergency_injector/mannitol = 2,\
+	/obj/item/reagent_containers/emergency_injector/epinephrine,\
 	/obj/item/device/analyzer/healthanalyzer)
+	
+/obj/item/storage/firstaid/crit
+	name = "emergency critical-condition first aid"
+	icon_state = "berserk1"
+	item_state = "berserk1"
+	desc = "An all-in-one emergency crash-kit designed to bring a critically wounded patient back from the brink of death. Contains cardiac stimulants, cerebral anti-edemics, and blood-oxygenation agents, all in convenient auto-injector form. Also contains an upgraded health analyzer to determine the health of the patient."
+	kit_styles = list("berserk1", "berserk2", "berserk3")
+	spawn_contents = list(/obj/item/reagent_containers/emergency_injector/atropine = 1,\
+	/obj/item/reagent_containers/emergency_injector/perf = 1,\
+	/obj/item/reagent_containers/emergency_injector/saline,\
+	/obj/item/reagent_containers/emergency_injector/mannitol = 2,\
```

Comment:
in light of recent changes, replacing the 2 mannitol injectors with a synaptizine injector and cutting the perf (atro+epi handles oxy well enough) might be better. Perhaps fill the space with one of each styptic and sulf patches

Reply:
Done! Though instead of patches, they get auto-menders.